### PR TITLE
[Merged by Bors] - chore(MeasureTheory/Integral/SetIntegral): rename type variables

### PR DESCRIPTION
**Pull request by grunweg**
Rename measurable spaces \alpha and \beta to X and Y.
Rename variables a : X and b : Y to x and y, respectively (and associated hypotheses as well).

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/MeasureTheory/Integral/SetIntegral.lean [hunk 1]
```diff
@@ -477,39 +477,39 @@ theorem integral_indicatorConstLp [CompleteSpace E]
...
+theorem set_integral_map {β} [MeasurableSpace β] {g : X → β} {f : β → E} {s : Set β}
     (hs : MeasurableSet s) (hf : AEStronglyMeasurable f (Measure.map g μ)) (hg : AEMeasurable g μ) :
     ∫ y in s, f y ∂Measure.map g μ = ∫ x in g ⁻¹' s, f (g x) ∂μ := by
   rw [Measure.restrict_map_of_aemeasurable hg hs,
     integral_map (hg.mono_measure Measure.restrict_le_self) (hf.mono_measure _)]
   exact Measure.map_mono_of_aemeasurable Measure.restrict_le_self hg
 #align measure_theory.set_integral_map MeasureTheory.set_integral_map
 
-theorem _root_.MeasurableEmbedding.set_integral_map {β} {_ : MeasurableSpace β} {f : α → β}
+theorem _root_.MeasurableEmbedding.set_integral_map {β} {_ : MeasurableSpace β} {f : X → β}
```
Comment (loefflerd) [inline Mathlib/MeasureTheory/Integral/SetIntegral.lean:488]: This lemma now involves two measurable spaces of which one is called `X` and the other `β`.

**Comment by loefflerd** [comment]
Could you please explain why you consider this change to be beneficial?

File: Mathlib/MeasureTheory/Integral/SetIntegral.lean (hunk 1 above)
Comment (grunweg) [reply Mathlib/MeasureTheory/Integral/SetIntegral.lean:488]: I'm happy to rename \beta to Y also, if that's desired.

**Comment by grunweg** [comment]
Sure! In short, as I understand it, using Greek letters dates back to 2017, but often differs from modern conventions.
(For topological spaces, this was explicitly confirmed on zulip; I have renamed a fair number of such types.)

For this file, many instances of \alpha are in fact topological spaces. As such, I think \alpha is a bad name (and X more standard).
I would think the same holds for measurable spaces; I'm open to switching to a different name (such as $\Omega$)!

File: Mathlib/MeasureTheory/Integral/SetIntegral.lean (hunk 1 above)
Comment (loefflerd) [reply Mathlib/MeasureTheory/Integral/SetIntegral.lean:488]: We definitely want the naming be consistent, so either both names should change or neither.

**Comment by grunweg** [comment]
awaiting-review

**Comment by grunweg** [comment]
Thanks for the careful look! I agree with all your comments.
I've gone over the entire file again and also tried to rename every single variable. Let me know if I missed one :-)

**Comment by loefflerd** [comment]
In `SetIndicatorConstLp` and `integral_indicatorConstLp`: rename `a` to `x` (and rename `x` to something else in the latter)

In `integrableOn_iUnion_of_summable_integral_norm` and around: indexing set shouldn't be `Y` (maybe iota?)

That's all I spotted. Thanks for fixing this!

**Comment by grunweg** [comment]
Well spotted! (I had wondered about the former, but not seen the latter.) Changed now.
awaiting-review

**Comment by loefflerd** [comment]
What's going on with the argument order in your last commit? I'm puzzled how changing the names of variables can make them reorder themselves!

**Comment by grunweg** [comment]
Along with renaming the variable, there's an implicit change which variable is being re-used.
Previously, Y was being re-used (which was declared at the beginning, on line 61); now, I'm using a new type \iota (which is declared on line 814). This causes the order change.

I just tweaked the variables on line 814 (putting iota first), but that's all I can do - short of putting \iota on line 61 also.

**Comment by loefflerd** [comment]
The change to variable order isn't a problem per se, I just wanted to check that it wasn't an indication of some unintentional code change. I'm labelling this as "awaiting CI" and if it passes then I'll flag it for a maintainer merge.

**Comment by loefflerd** [comment]
maintainer merge

**Review by jcommelin — APPROVED** [review]
Thanks :tada:

bors merge

**Pull request closed without merging**